src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -199,139 +199,154 @@ export const BaseComment = ({
     return (
         // TODO: Change className to bcs-Comment once FF is removed
         <div className="bcs-BaseComment">
-            <Media
+            <div
                 className={classNames('bcs-Comment-media', {
                     'bcs-is-pending': isPending || error,
                 })}
             >
-                <Media.Figure>
-                    <Avatar
-                        getAvatarUrl={getAvatarUrl}
-                        user={createdByUser}
-                        badgeIcon={
-                            annotationActivityLink && (
-                                <IconAnnotation title={<FormattedMessage {...messages.annotationBadge} />} />
-                            )
-                        }
-                    />
-                </Media.Figure>
-                <Media.Body>
-                    {isMenuVisible && (
-                        <TetherComponent
-                            attachment="top right"
-                            className="bcs-Comment-deleteConfirmationModal"
-                            constraints={[{ to: 'scrollParent', attachment: 'together' }]}
-                            targetAttachment="bottom right"
-                        >
-                            <Media.Menu
-                                isDisabled={isConfirmingDelete}
-                                data-testid="comment-actions-menu"
-                                dropdownProps={{
-                                    onMenuOpen: () => onSelect(true),
-                                    onMenuClose: handleMenuClose,
-                                }}
-                                menuProps={{
-                                    'data-resin-component': ACTIVITY_TARGETS.COMMENT_OPTIONS,
-                                }}
+                <div className="bcs-BaseComment-MainComment">
+                    <div className="bcs-BaseComment-MetaRow">
+                        <div className="bcs-BaseComment-MetaRow-Metadata">
+                            <div className="bcs-BaseComment-MetaRow-Metadata-Avatar">
+                                <Avatar
+                                    getAvatarUrl={getAvatarUrl}
+                                    user={createdByUser}
+                                    badgeIcon={
+                                        annotationActivityLink && (
+                                            <IconAnnotation
+                                                title={<FormattedMessage {...messages.annotationBadge} />}
+                                            />
+                                        )
+                                    }
+                                />
+                            </div>
+                            <div className="bcs-BaseComment-MetaRow-Metadata-Data">
```

Comment:
We seem to have delved too greedily and too deep with the BaseComment component. The nesting is getting a bit intense and the naming is losing its meaning. It looks like we may need to refactor into a few smaller components at this point.

Reply:
I 100% agree. Sorry I haven't added a PR description yet. I copied it out while I was making some changes, and forgot to copy it back in, but in the original description I said this:

> Although a re-architecture of BaseComment may be in order, for the purposes of this task, I tried to constrain myself to doing the minimum required to implement the [new layout](https://www.figma.com/file/EyW24R6Bg9IkiQ33urle5l/Preview-Threaded-Replies-and-Annotations?node-id=1169%3A189350&mode=dev) of comments and annotations.

The reason I say this is because I have already proposed re-architecturing of BaseComment long ago but it was not perceived to be worth it at the time. I mentioned this to Adam in a DM on Monday to get his advice, he advised to proceed as is, and try proposing re-architecturing again in-between tasks. That sounded wise to me. What d'you think?

Reply:
Re-architecture and refactoring are sometimes different. I haven't seen the prior proposal, so I can't say. In broad terms, we should take the time to build things the best way we can within reason. I would imagine extracting the new layout to its own component(s), along with maybe the menu, would help a lot at relatively low cost.

Reply:
That sounds pretty good. We've discussed just that refactor. I think that's a good place to start, and I'd be happy to do it in this PR. 

I also recommended [this re-architecture](https://github.com/box/box-ui-elements/pull/3321) back in the day. Probably the key point there is solving the circular dependency on Replies by passing it in as a child. That's a more major overhaul. Do you think it's worth brining that up again?

Reply:
The overall architecture as a conept is ok but we can break BaseComment down into constituent components. Right now we just want to make sure that everything is ready for the Boxer release because I would be shocked if we don't have more changes to make after people start using it.

Reply:
Maybe the overall architecture is okay, but certainly not great. It would certainly be better to define the Replies component in its own file rather than defining it in BaseComment.js. But I also see that it might not be worth it. Super happy about the green light for re-organizing into constituent components though, and I will get on top of that right away :)